shoes-core/lib/shoes/line.rb

```diff
@@ -19,6 +21,25 @@ def create_dimensions(point_a, point_b)
       style[:y2] = point_b.y
     end
 
+    def in_bounds?(x, y)
```

Comment:
A link to the page where you "stole" the algorithm from would be nice as a comment above the method :)